lte/gateway/c/session_manager/LocalSessionManagerHandler.cpp

```diff
@@ -246,35 +244,30 @@ void LocalSessionManagerHandlerImpl::send_create_session(
         PrintGrpcMessage(
             static_cast<const google::protobuf::Message&>(response));
         if (status.ok()) {
-          bool success = enforcer_->init_session_credit(
+          enforcer_->init_session_credit(
               *session_map_ptr, imsi, sid, cfg, response);
-          if (!success) {
-            MLOG(MERROR) << "Failed to initialize session for IMSI " << imsi;
-            status = Status(
-                grpc::FAILED_PRECONDITION, "Failed to initialize session");
+
+          std::string bearer_id = "";
+          if (cfg.rat_specific_context.has_lte_context()) {
```

Comment:
maybe that logic would look nicer inside the SessionConfig

```
...
  std::experimental::optional<AggregatedMaximumBitrate> get_apn_ambr() const;
  std::string get_bearer_id_if_any();
};

std::string SessionConfig::get_bearer_id_if_any(){
  std::string bearer_id = "";
  if (rat_specific_context.has_lte_context()){
    bearer_id = rat_specific_context.lte_context().bearer_id();
  }
  return bearer_id;
}
```

So you can hide that and just add `cfg.get_bearer_id_if_any()` to your log calls